clients/ts/signalr/tests/HubConnection.test.ts

```diff
@@ -1029,21 +1041,22 @@ class TestConnection implements IConnection {
 
     public receive(data: any): void {
         const payload = JSON.stringify(data);
-        this.onreceive(TextMessageFormat.write(payload));
+        this.invokeOnReceive(TextMessageFormat.write(payload));
     }
 
     public receiveText(data: string) {
-        this.onreceive(data);
+        this.invokeOnReceive(data);
     }
 
     public receiveBinary(data: ArrayBuffer) {
-        this.onreceive(data);
+        this.invokeOnReceive(data);
     }
 
-    public onreceive: (data: string | ArrayBuffer) => void;
-    public onclose: (error?: Error) => void;
-    public sentData: any[];
-    public lastInvocationId: string;
+    invokeOnReceive(data: string | ArrayBuffer) {
```

Comment:
We're probably turning TSLint on for tests soon, and it will get grumpy if this doesn't say `private` ;)